src/test/java/com/wikia/webdriver/elements/oasis/pages/WikiFeatures.java

```diff
@@ -0,0 +1,63 @@
+package com.wikia.webdriver.elements.oasis.pages;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class WikiFeatures extends BasePageObject {
+
+  @FindBy(css = "#WikiFeatures > ul:nth-child(3) > li:nth-child(1) > div.actions > span > span.button")
+  private WebElement europaInfoboxThemeSlider;
+
+  @FindBy(css = "#WikiFeatures > ul:nth-child(3) > li:nth-child(1) > div.actions > span > span.texton")
+  private WebElement europaInfoboxThemeSliderText;
+
+  @FindBy(css = "#DeactivateDialog")
+  private WebElement deactivateDialog;
+
+  @FindBy(css = "#DeactivateDialog > footer > div > button.button.normal.primary")
+  private WebElement confirmDeactivation;
+
+  @FindBy(css = ".feature")
+  private List<WebElement> features;
+
+  public WikiFeatures() { super(); }
+
+  public WikiFeatures openWikiFeatures(String wikiURL) {
+    getUrl(wikiURL + URLsContent.SPECIAL_WIKI_FEATURES);
+    PageObjectLogging.log("openWikiFeaturesPage", "special wiki features page opened", true, driver);
+
+    return this;
+  }
+
+  public boolean isEuropaInfoboxThemeEnabled() {
+    if (europaInfoboxThemeSliderText.getText().equalsIgnoreCase("Enabled")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1126)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "Enabled" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)